core/src/main/java/bio/terra/pearl/core/service/export/formatters/module/SurveyFormatter.java

```diff
@@ -90,6 +170,13 @@ public String getColumnHeader(ItemFormatter itemFormatter, boolean isOtherDescri
             AnswerItemFormatter answerItemFormatter = (AnswerItemFormatter) itemFormatter;
             // for now, strip the prefixes to aid in readability.  Once we have multi-source surveys, we can revisit this.
             String cleanStableId = stripStudyAndSurveyPrefixes(answerItemFormatter.getQuestionStableId());
+            if (Objects.nonNull(answerItemFormatter.getParentStableId())) {
+                cleanStableId = stripStudyAndSurveyPrefixes(answerItemFormatter.getParentStableId()) + "." + cleanStableId;
+            }
+            if (Objects.nonNull(answerItemFormatter.getRepeatIndex())) {
+                cleanStableId += "[" + answerItemFormatter.getRepeatIndex() + "]";
```

Comment:
use the helper method you created above.  In general, try to get a single source of truth for how these names are encoded